src/components/CohortBuilder/ParticipantsTableView/ParticipantsTable.js

```diff
@@ -133,6 +135,32 @@ const CellNbFilesCell = (value, row) => <NbFilesCell value={value} row={row} />;
 
 const CellParticipantIdLink = (value) => <ParticipantIdLink value={value} />;
 
+const getIt = (participant, accessor) => get(participant, accessor, null);
+
+const CellPedcBioLink = (props) => {
+  if (STUDIES_WITH_PEDCBIO.includes((props.object || {}).studyId)) {
+    return (
+      <ExternalLink
+        href={`https://pedcbioportal.kidsfirstdrc.org/patient?studyId=pbta_all&caseId=${getIt(
+          props.object,
+          props.accessor,
+        )}`}
+        onClick={() => {
```

Comment:
I think that the curly braces are useless plus I think that it's a mistake (not yours already like that in the code) not to make "async/await" before trackUserInteraction